src/createAnimatedComponent.js

```diff
@@ -197,8 +197,12 @@ export default function createAnimatedComponent(Component) {
       const style = {};
       for (const key in inputStyle) {
         const value = inputStyle[key];
-        if (!(value instanceof AnimatedNode) && key !== 'transform') {
-          style[key] = value;
+        if (key !== 'transform') {
+          if (value instanceof AnimatedNode) {
+            style[key] = value.__getValue();
+          } else {
+            style[key] = value;
+          }
```

Comment:
You could probably use ternary operator here, but I am not sure if it would be as readable as the code above 🤔

Reply:
Thought about it but this is a bit long condition for my taste and decided that plain if is more readable